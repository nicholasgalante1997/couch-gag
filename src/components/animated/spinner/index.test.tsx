import { Spinner } from './index';
import { render } from '@testing-library/react';

describe('components/animated/spinner', () => {
    test('spinner/default', () => {
        const { queryByTestId } = render(<Spinner data-testid="spinner-test" />);
        const spinner = queryByTestId('spinner-test');
        expect(spinner).toBeInTheDocument();
        expect(spinner).toMatchSnapshot();
    });
});