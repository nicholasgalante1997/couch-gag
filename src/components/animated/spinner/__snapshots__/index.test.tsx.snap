// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/animated/spinner spinner/default 1`] = `
<div
  class="spinner"
  data-testid="spinner-test"
/>
`;
